cartridge-cli.lua

```diff
@@ -2727,7 +2727,13 @@ end
 
 local function tarantool_repo_version()
     local parts = _TARANTOOL:split('.')
-    return string.format('%s_%s', parts[1], parts[2])
+    local version = string.format('%s_%s', parts[1], parts[2])
+
+    if version == '2_1' then
```

Comment:
may be comment?

Reply:
Test is the best variant (of course). But it leads to adding new target to CI

Reply:
Yes, I will add a comment.

I was thinking about unit test, but I realized that changing `_G._TARANTOOL` isn't good idea.
Anyway, I think that comment will help to prevent changing this code.